deployment/docker-compose.template.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
```

Comment:
I'd appreciate a comment at the top line explaining why this file is here

Reply:
This file is the way to deploy to ecs. This whole directory is the deployment scripts required to put the code in aws.

Reply:
Added a comment and a link to related aws documentation.